torch/csrc/jit/script/compiler.cpp

```diff
@@ -1632,7 +1632,15 @@ struct to_ir {
   // Get the appropriate builtin op for this augmented assignment
   // If the RHS is a tensor, return the corresponding ATen in-place op
   // If it's a list of scalars, then return the corresponding list augment op
-  Symbol getAugOp(const AugAssign& stmt, bool isTensor) {
```

Comment:
not sure what you're doing here, this seems unrelated. Can we have this in a separate PR?

Reply:
This is what's needed to make `a += b` do the inplace add instead of desugaring it into `a = a + b`.